src/components/mixins/validation.js

```diff
@@ -98,12 +98,12 @@ export const _amount = (wallet) => (amount) => {
 }
 
 export const _explorerQuery = (type) => (query) => {
-  if (type === SearchTypes.ACCOUNT_TYPE) {
+  if (type === SearchTypes.ACCOUNT) {
     return _user.nameDomain(query)
-  } else if (type === SearchTypes.BLOCK_TYPE) {
+  } else if (type === SearchTypes.BLOCK) {
     const parsed = parseInt(query)
     return Number.isInteger(parsed) && parsed.toString().length === query.length
```

Comment:
It's not related to this PR but what are you doing here? If you're checking for is query is number it can be done much simpler. If I understand this validation correctly, it can be done by `Number.isInteger(Number(query))`